crates/red_knot_python_semantic/src/builtins.rs

```diff
@@ -0,0 +1,12 @@
+use red_knot_module_resolver::resolve_builtins;
+
+use crate::semantic_index::module_global_scope;
+use crate::{semantic_index::symbol::ScopeId, Db};
+
+#[salsa::tracked]
+pub(crate) fn builtins_scope(db: &dyn Db) -> Option<ScopeId<'_>> {
```

Comment:
What's the reason for returning an option if it is always some

Reply:
It's not always `Some`. There's a question-mark operator in there.

Reply:
There's a question mark in the function, so it's not always Some